libs/core/src/lib/list/list-navigation-item/list-navigation-item.component.html

```diff
@@ -0,0 +1,33 @@
+<ng-container *ngIf="!expanded || !_isExpandable || !_condensed || !_listComponent">
+    <ng-content></ng-content
+    ><span *ngIf="indicated || _childIndicatedAndCollapsed()" class="fd-list__navigation-item-indicator"></span>
+</ng-container>
+<ng-container *ngIf="expanded && _isExpandable && _condensed && _listComponent">
+    <div
+        class="fd-popover__body fd-popover__body--no-arrow fd-list__navigation-item-popover--first-level"
+        aria-hidden="false"
+    >
+        <ul class="fd-list">
+            <li class="fd-list__navigation-item fd-list__navigation-item--condensed">
+                <fd-icon class="fd-list__navigation-item-icon" [glyph]="_iconComponent.glyph"></fd-icon>
+                <span class="fd-list__navigation-item-text">{{ _innerText }}</span>
+            </li>
+        </ul>
+    </div>
+    <div
+        class="fd-popover__body fd-popover__body--no-arrow fd-list__navigation-item-popover--second-level"
+        aria-hidden="false"
+    >
+        <ul class="fd-list">
+            <li
+                *ngFor="let navItem of _listComponent._navItems"
+                [ngClass]="{ 'fd-list__navigation-item--indicated': navItem.indicated }"
```

Comment:
```suggestion
                [сlass.fd-list__navigation-item--indicated]="navItem.indicated"
```

Reply:
Why? My IDE doesn't like this syntax
![Screen Shot 2021-09-27 at 10 49 49 AM](https://user-images.githubusercontent.com/2471874/134951821-322207be-1b9d-434d-92e7-afebab3b9f4d.png)

Reply:
`[class]` preferred over the `[ngClass]`
https://angular.io/guide/built-in-directives#adding-and-removing-classes-with-ngclass
https://angular.io/guide/attribute-binding#binding-to-multiple-css-classes

But it behaves something differently:
https://github.com/angular/angular/issues/40623
https://hackmd.io/jzDc7hIDTdWtQblv2TbL9A